### ISSUE TITLE
Configuration for rule "indent" is invalid: Value "data["1"].MemberExpression has additional properties

### ISSUE THREAD

**Issue by bensaufley** [Microsoft/vscode-eslint#389]
I'm getting this popping up over my editor whenever I open a JS file. Running `./node_modules/.bin/eslint path/to/that/file.js` within the project root does not produce this error.

ESLint version: 4.14.0
Plugin version: 1.4.3
VS Code version: 1.19.2

I don't see a `data["1"]` anywhere; the only reference to `MemberExpression` I'm finding (and the only one I'm finding in the ESLint docs) is in `indent`, which looks like this in my `.eslintrc.json`:

```json
    "indent": [
      "warn",
      2,
      {
        "VariableDeclarator": {
          "var": 2,
          "let": 2,
          "const": 3
        },
        "MemberExpression": 1,
        "SwitchCase": 1
      }
    ]
```

Please let me know what else I can tell you.

**Comment by dbaeumer** [comment]
Can you provide me with a GitHub repository I can clone that demos the problem.

**Comment by ChrisYhP** [comment]
i have the same problem....how to fix it

**Comment by Tonysmark** [comment]
Same problem bro...

**Comment by dbaeumer** [comment]
This is a problem with the `.eslintrc.json` file what happens when you run eslint from a terminal using the same configuration.

**Comment by egucciar** [comment]
Same problem ....sad!!!

**Comment by dbaeumer** [comment]
Actually I can't reproduce this. Here is what I get:

<img width="935" alt="capture" src="https://user-images.githubusercontent.com/1931590/38980773-015dcbda-43be-11e8-9975-9eed0a4316a4.png">

Someone who can reproduce this can you please provide me with a GitHub repository that demos this.

**Comment by egucciar** [comment]
Yeah agree I got it working too. Weird, it turned out to be the Switch Case as Pascal and I had it as camel. Sorry! But the error msg wasn't helping

**Comment by dbaeumer** [comment]
The error message comes from the npm module. To improve it please file an issue against https://github.com/eslint/eslint

OK to close ?

**Comment by egucciar** [comment]
I believe my issue only looked like the issue in the thread. Note the other users switch case case is alright.

**Comment by bensaufley** [comment]
FWIW I've moved to `eslint-config-airbnb` in most projects, and updated eslint several times at that, so whether or not this has been fixed I'm no longer encountering it. The issue was in a private repo. I'm not sure if I tried recreating it in a public example repo. I'm fine with closing; if I come across it again I'll try to create a public example repo and ask to reopen.
